### ISSUE TITLE
preventEventOnTouchMove prevent vertical scroll of the page

### ISSUE THREAD

**Issue by felixmosh** [maxmarinich/vanilla-swipe#4]
Hi,

I have a list of carousels in mobile view.
When page scroll starts at the carousel it got prevented, there is any way to allow it?

**Comment by maxmarinich** [comment]
Hi, @felixmosh! Not sure if I understood your question correctly, but you can't switch `preventDefaultTouchmoveEvent` property on the fly. If you set it in `true` during initialization, the only way to change it is to call the `update` method. Also need to consider, that `Event` can be prevented if it has `cancelable` property in `true`. Condition [link](https://github.com/maxmarinich/vanilla-swipe/blob/3fb34a7640fa17178329c97a9c306d16b42606fe/src/index.ts#L129). 

Hope i answered your question. If not, please let me know.

**Comment by felixmosh** [comment]
Thanx for the quick answer, 

I'm using alice-react-carousel that uses this lib, as I've described, when I enable this flag, I can't scroll my **page** vertically if the scroll starts at one of the carousels.

**Comment by maxmarinich** [comment]
Probably, you not set mobile [mod](https://drive.google.com/file/d/1ikyLoNwEvuQ3r75Lo461CCpFcE4L_Ko-/view). Please [check](https://zrlmr3p61m.csb.app/).

**Comment by felixmosh** [comment]
I've tested it on my mobile phone :]
It is reproduces also in mobile mode in chrome.

I've updated your example to reproduce it
https://74kpx.csb.app/

**Comment by maxmarinich** [comment]
Probably, the issue is for other library. And i guess you set a property not correctly for the carousel(`preventEventOnTouchMove === true`). To avoid preventing it should be `false`. Please [check](https://drive.google.com/file/d/1Oi2d-F5oKKV0_c5iKOaxEgTlozynIeYz/view).

**Comment by develohpanda** [comment]
Hi @maxmarinich! Brilliant library - `react-alice-carousel` is very nice, however I am having the same issue as @felixmosh. 😢 

If I set `preventEventOnTouchMove === true`, then I cannot scroll the **page** if the scroll is initiated on a carousel. If I set `preventEventOnTouchMove === false`, then the page and carousel both scroll at the same time.

We need the best of both worlds. IMO, preventing the event on touchMove should only happen when the swipe direction is horizontal, and should **not** prevent if a vertical scroll is initiated on a carousel.

https://github.com/maxmarinich/vanilla-swipe/blob/3fb34a7640fa17178329c97a9c306d16b42606fe/src/index.ts#L127-L129

Let me know if I can help with a fix.

😃
